CMakeLists.txt

```diff
@@ -108,7 +108,7 @@ endif()
 
 set(UHTTP_C_INC_FOLDER ${CMAKE_CURRENT_LIST_DIR}/inc CACHE INTERNAL "this is what needs to be included if using sharedLib lib" FORCE)
 
-set(use_cppunittest ON)
+set(use_cppunittest OFF)
```

Comment:
Why are these turned off?

Reply:
There are issues in dependent repos. This stops running the unit tests using the C++ compiler. Now they build using the C compiler.